### ISSUE TITLE
Custom "from" email for campaign

### ISSUE THREAD

**Issue by kgirzadas** [responsiv/campaign-plugin#39]
It would be nice to have ability to optionally set "from" email for each campaign.

**Comment by daftspunk** [comment]
Agreed this could be a handy feature. Would you mind sharing more details on how you would use it yourself? This will be used to formulate a solution.

**Comment by kgirzadas** [comment]
In my case, global "from" email in mail configuration is set to `info@domain.com`. It is used for contact forms submissions. Our newsletter campaigns target is replies and not traffic to our website. So for newsletter campaigns we would like to use different email address, for example - `newsletter@domain.com`. And for big campaigns we would like to set different email address per each of them to have control on flow of emails.

So, it would be nice to have:
**1. Optional global "from" email setting for campaign plugin.** 
If set it will be used for all emails from this plugin. 

**2. Optional "from" email setting for each campaign.** 
If set it will be used for all campaign emails.

**Comment by Fl0Cri** [comment]
+1, I just had exactly the same situation while migrating from a Joomla site that was using AcyMailing.

I am also wanting to send my mails with an address like `web@example.com` for all diverse mails like form confirmations, and have a `newsletter@example.com` that will be used to send my newsletters.

In AcyMailing the mail output (including sending method like smtp, sendmail, mailchimp, etc) was defined within the AcyMailing config, so it doesn't use the mail config of the CMS. The only thing that I ever had to change between both outputs were the sender address and name.

I like @Kunilt solution for the possibility to define per campaign setting.

It would be great to have list that could be named `Senders` and where we could define multiple settings such as `from email`, `from name`, and maybe other settings in the future. We then could select a "sender" from a campaign or define a default "sender" in the plugin config.

**Comment by daftspunk** [comment]
Thanks for the feedback!

**Comment by ChVuagniaux** [comment]
+1 
It's could be very nice to can have multi senders configuration
A default that use site configs : sender name, sender email (like now)
and can create many others with Name, Email From, Reply-To

That we can choose when creating new Campaigns.

**Comment by chocolata** [comment]
Hi, just came here to back this feature. I'm setting up a multisite project for a client. They have five subcompanies each with their own site, all of them having slightly different styling. This is perfectly possible in October CMS thanks to the multisite feature, but problems arise when sending out a mail campaign, since at the moment we can only send out with one sender.

If we could set the sender email and name manually per campaign, this would solve the problem.

We could set up a list and custom template for each brand, but setting the sender name and email for each campaign would be the final piece in the puzzle...

Could you kindly consider adding this feature?
Thanks.

**Comment by chocolata** [comment]
I think this is solved with the new multisite feature?
